airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSourceOperations.java

```diff
@@ -60,6 +63,13 @@ public class PostgresSourceOperations extends AbstractJdbcCompatibleSourceOperat
   private static final String TIMESTAMPTZ = "timestamptz";
   private static final String TIMETZ = "timetz";
   private static final ObjectMapper OBJECT_MAPPER = MoreMappers.initMapper();
+  private static final Map<Integer, PostgresType> POSTGRES_TYPE_DICT = new HashMap<>();
+  private Map<String, Map<String, ColumnInfo>> streamColumnInfo = new HashMap<>();
+
+  public PostgresSourceOperations() {
+    super();
+    Arrays.stream(PostgresType.class.getEnumConstants()).forEach(c -> POSTGRES_TYPE_DICT.put(c.type, c));
```

Comment:
I think this is weird that you are doing in the constructor beacause you are filling up a global static. Maybe multiple times.
does this work in Java?
```java
// at the top where it is now.
private static final Map<Integer, PostgresType> POSTGRES_TYPE_DICT = new HashMap<>(); 
static
    {
        Arrays.stream(PostgresType.class.getEnumConstants()).forEach(c -> POSTGRES_TYPE_DICT.put(c.type, c));
    }
```

If it has to be in a constructor, then at least make sure you are only doing it once by it not being empty or with a boolean.